src/dotnet/HQ.Server/Controllers/TimeEntriesControllerV1.cs

```diff
@@ -66,17 +74,7 @@ public async Task<ActionResult> UpsertTime1([FromBody] UpsertTimeV1.Request requ
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
         public async Task<ActionResult> GetTimesV1([FromBody] GetTimesV1.Request request, CancellationToken ct = default)
         {
-            var staffId = User.GetStaffId();
-            var staff = await _context.Staff
-                .AsNoTracking()
-                .SingleOrDefaultAsync(t => t.Id == staffId);
-
-            if(staff == null)
-            {
-                return NotFound();
-            }
-
-            request.StaffId = staffId;
+            request.StaffId = User.GetStaffId();
```

Comment:
When getting time entries we dont want to force the staff ID in the controller, staff should be allowed to query for time from other staff members. 

In the CLI and UI (when we get designed) we will default it to the current logged in user's staff ID.